src/js/components/form/FieldTextarea.js

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames/dedupe';
+import React from 'react';
+
+import FieldInput from './FieldInput';
+import Util from '../../utils/Util';
+
+class FieldTextarea extends FieldInput {
+  render() {
+    let {
+      error,
+      formGroupClass,
+      formGroupErrorClass
+    } = this.props;
+    let attributes = Util.omit(this.props, Object.keys(FieldInput.propTypes));
+
+    let classes = classNames(
+      {[formGroupErrorClass]: !!error},
+      formGroupClass
+    );
+
+    return (
+      <div className={classes}>
+        {this.getLabel()}
+        <textarea {...attributes} />
```

Comment:
Let's do a this.getElement(attributes) helper function. Which we do overwrite in the FieldTextarea.

so 
```JS
getElement(attributes) {
  return (
    <textarea {...attributes} />
  );
}
```

would be used in render as `{this.getElement(attributes)}` 

and we only need to overwrite the getElement function rather then the `render`, as far as I see this only the element is the only difference in the render function compared with the FieldInput. we are even omitting the props based on `FieldInput.propTypes`.

Another solution, which is more composable, is that we have a wrapping component which does contain all the help block, error block and labels and does also await children which then could be a input or a textarea

e.g.:
```HTML
<FormElementWrapper>
  <input type="text" />
</FormElementWrapper>